Samples/Excel-Add-in-ChartAPI-Analysis-Data/src/commands/commands.html

```diff
@@ -0,0 +1,18 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. -->
```

Comment:
Please remove command files from projects that don't need them.

Reply:
Done